src/functionalTest/resources/features/F-004 - Put Dictionary /F-004.feature

```diff
@@ -0,0 +1,127 @@
+    Then  a negative response is received
+    And   the response [has the 403 Forbidden status code]
+    And   the response has all other details as expected.
+
+#-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+  @S-004.5 #WLTS-4 AC5 - Translation is empty
+  Scenario: Bad Request - Return 400 error when Translations are empty
+
+    Given a user [with manage-translation IDAM role],
+    When  a request is prepared with appropriate values,
+    And   the request [where the translations are empty]
+    And   it is submitted to call the [PUT dictionary] operation of [Translation Service]
+    Then  a negative response is received
+    And   the response [has the 400 Bad Request status code]
+    And   the response has all other details as expected.
+
+#-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+  @S-004.6 #AC5 - Translation is null
+  Scenario: Bad Request - Return 400 error when Translations are null
+
+    Given a user [with manage-translation IDAM role],
+    When  a request is prepared with appropriate values,
+    And   the request [where the translations are null]
+    And   it is submitted to call the [PUT dictionary] operation of [Translation Service]
+    Then  a negative response is received
+    And   the response [has the 400 Bad Request status code]
+    And   the response has all other details as expected.
+
+#-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+
+  @S-004.7 #WLTS 4 AC6
+  Scenario: Welsh not allowed for this user - Return 400 error
+
+    Given a user [without manage-translation IDAM role],
+    When  a request is prepared with appropriate values,
+    And   the request [has welsh phrases]
+    And   it is submitted to call the [PUT dictionary] operation of [Translation Service]
+    Then  a negative response is received
+    And   the response [has the 400 Bad Request status code]
+    And   the response has all other details as expected.
+
+#-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+
+  @S-004.8 #WLTS-20 AC1
+  Scenario: Do nothing and return success - when service is definition store
+
+    Given a user [without manage-translation or load-translation IDAM role and uses ccd-definition],
+    And   a successful call [to PUT translation phrases into the dictionary] as in [S-004.8_Existing_Data],
+    When  a request is prepared with appropriate values
```

Comment:
Can we add an extra spec from the AC to explanin why it is doing nothing: e.g. 
```
AND the request [contain an entry for an english_phrase that already exists] 
```

Reply:
done